src/actions/dataset.ts

```diff
@@ -81,7 +84,9 @@ export function datasetLoad(name: string, dataset: Data): DatasetLoad {
     } else {
       throw new Error('dataset load error: dataset type not detected');
     }
-
-
   };
 };
+
+function errorCatch(err: any) {
```

Comment:
Don't Typescript already provide `Error` type or something.  (We should avoid `any`.)